pkg/scheduler/apis/config/validation/validation_test.go

```diff
@@ -136,6 +140,10 @@ func TestValidateKubeSchedulerConfiguration(t *testing.T) {
 			expectedToFail: true,
 			config:         bindTimeoutUnset,
 		},
+		"bad-percentage-of-nodes-to-score": {
+			expectedToFail: true,
+			config:         percentageOfNodesToScore101,
```

Comment:
Probably a positive test case as well?

Reply:
The positive test case exists. It is labeled as "good" in the switch statement. The number 35 that I have added above is for the valid case.

Reply:
Ohh sorry github UI wasn't showing as it wasn't a new change, so did not notice. Thanks.